kaggle/api/kaggle_api_extended.py

```diff
@@ -542,7 +508,7 @@ def _load_config(self, config_data):
     self.config_values = config_data
 
     try:
-      self.api_client = ApiClient(configuration)
+      self.api_client = None  # ApiClient(configuration)

```

Comment:
Is this intentional?  If so, when is `api_client` filled in (or should it be removed)?

Reply:
Removed, thanks.